### PR TITLE
Show list of XHR request URLs in popup

### LINKED ISSUE #4: Show the list of detected XHR requests in the popup
When interceptor is enabled for a browser tab, all XHR requests should be logged and listed in the popup UI

### PR DESCRIPTION
**Pull request by emilsoman**
Closes #4

### CODE REVIEW

File: app/content_script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { requestStore } from './store'
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if(request.message == "LOG_REQUEST"){
```
Comment (kgrz) [inline app/content_script.js]: is `===` better here?

File: app/background.js [hunk 2]
```diff
@@ -36,6 +39,7 @@ class BackgroundWorker {
       (details) => {
         this.data[tabId].count += 1;
         chrome.browserAction.setBadgeText({text: '' + this.data[tabId].count, tabId: tabId});
```
Comment (kgrz) [inline app/background.js]: This is not modified in this PR, but interpolation would be better IMO. Babel has support afaik

Comment (emilsoman) [inline app/background.js]: What I'm doing here is convert an integer into a string using the old trick `'' + integer` . Do you mean to say do this instead? : `` `${integer}` ``

Comment (kgrz) [inline app/background.js]: yeah, that's what I meant

**Comment by emilsoman** [comment]
Closing this because I thought of a better way to show URLs . See #12 .

Problem with this PR is the following:

A popup's lifespan starts when you click on the extension's browser button and stops when you close it.
This means you cannot persist stuff in the markup. Another problem is that the popup cannot listen to messages from the backend worker script.

Moving this to a content script makes sense because a content script will be alive as long as the tab is open, and it can listen to events from the backend worker - which means we can show a live updating list of XHR urls.

**Pull request closed without merging**